lib/extended_bundler/errors.rb

```diff
@@ -68,6 +72,37 @@ def troubleshoot(spec_install)
 
       private
 
+      def update_handlers
+        puts "[ExtendedBundler] Updating Extended Bundler Errors Index"
+        handlers = Cache.handlers_index
+        return if handlers.nil?
+
+        handlers_to_update = []
+        handlers.select do |entry|
+          file, mtime = entry.split(',')
+          # Handlers are up to date if they exist and the mtime hasnt changed on master
+          # Otherwise, we need to update them locally
+          up_to_date = File.exist?(file) && File.mtime(file).utc >= Time.parse(mtime)
```

Comment:
Is this relative path definitely looking in the right place?

Reply:
yep! it should be, but I can work on making it more explicit

Reply:
It was the use of `File.expand_path("../../cache/index_cache_time", __dir__)` elsewhere that threw me.

Reply:
yea, ill make this nicer